ttnn/cpp/ttnn/operations/conv/conv2d/conv2d.hpp

```diff
@@ -32,14 +43,14 @@ Result conv2d_L1(
     uint32_t input_height,
     uint32_t input_width,
     std::array<uint32_t, 2> kernel_size,
-    std::array<uint32_t, 2> stride,
-    std::variant<std::array<uint32_t, 2>, std::array<uint32_t, 4>> padding,
-    std::array<uint32_t, 2> dilation,
-    uint32_t groups,
-    const std::optional<const ttnn::Tensor>& bias_tensor,
-    const std::optional<const Conv2dConfig>& conv_config_,
-    const std::optional<const DeviceComputeKernelConfig>& compute_config_,
-    const std::optional<const MemoryConfig>& memory_config);
+    std::array<uint32_t, 2> stride = std::array<uint32_t, 2>{1, 1},
```

Comment:
why do we need defaults on conv2d_L1, conv2d_DRAM, conv2d
isn't it enough to have them on invoke methods?